app/util/job.ts

```diff
@@ -12,20 +11,6 @@ import { getWorkflowStepsByJobId } from '../models/workflow-steps';
 import DataOperation, { CURRENT_SCHEMA_VERSION } from '../models/data-operation';
 import { createDecrypter, createEncrypter } from './crypto';
 
-/**
- * Cleans up the temporary work items for the provided jobID
- * @param jobID - the jobID for which to remove temporary work items
- * @param logger - the logger associated with the request
- */
-async function cleanupWorkItemsForJobID(jobID: string, logger: Logger): Promise<void> {
```

Comment:
Can you add a note to HARMONY-1189 that we are no longer cleaning up EFS, so if for any reason we can't implement HARMONY-1189 the way we expect we need to clean up EFS.

Reply:
Done.